src/content/development/contribution-guide/_index.en.md

```diff
@@ -78,11 +78,12 @@ To be expanded
 
 ### Unplanned Work
 
-To be expanded
-
 #### Ongoing Maintenance
 
-To be expanded
+Tasks that relate to ongoing maintenance (version bumps, image trimmings, CI and automation) will usually be small and unplanned
+tasks that we can't plan ahead for, but we know there will always be work to do.
+Use label "maintenance" on the issue.
```

Comment:
... and on PRs? (I’m thinking of dependabot PRs for example where we won’t have an issue filed before the PR is submitted.)